software/control/core/core.py

```diff
@@ -60,29 +60,31 @@
 
 class ObjectiveStore:
     def __init__(self, objectives_dict=OBJECTIVES, default_objective=DEFAULT_OBJECTIVE, parent=None):
+        self.parent = parent
         self.objectives_dict = objectives_dict
         self.default_objective = default_objective
         self.current_objective = default_objective
         self.tube_lens_mm = TUBE_LENS_MM
         self.sensor_pixel_size_um = CAMERA_PIXEL_SIZE_UM[CAMERA_SENSOR]
-        self.pixel_binning = self.get_pixel_binning()
-        self.pixel_size_um = self.calculate_pixel_size(self.current_objective)
+        self.pixel_size_um_unbinned = self._calculate_pixel_size(self.current_objective)
```

Comment:
@Alpaca233 in a future PR these and the related functions should be moved out of ObjectiveStore

Reply:
@Alpaca233 @ianohara perhaps we can discuss where we should move this to on slack.